hw/top_earlgrey/syn/chip_earlgrey_asic_check_only.sdc

```diff
@@ -0,0 +1,53 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# These SDC's are not used for synthesis, since they collide with the current
+# set_clock_groups -async constraints. Instead, they are used to time the design
+# after P&R in order to make sure that the CDC crossings are all within spec.
```

Comment:
@arnonsha this is the separate file for PT. The generic 2 FF max delay constraint should also apply to the differential alert channels, since these use the standard synchronizer primitive.

Reply:
Thanks, we'll try them out